File: PRAK602/PRAK602-2210817310015-RIDHANISETIADI.py
kolom = int(input())
angka = list(map (int, input().split()))
matriks = []
x = 0
for i in range(0, kolom) :
    matriks.append(angka[x : x + kolom])
for i in range(0, kolom) :
    print(matriks[0][i] * (i + 1),end=" ")

File: PRAK602/PRAK602-2210817310015-RIDHANISETIADI.c
#include<stdio.h>
int main(void){
    int kolom,isi,a;
    scanf("%d",&kolom);
    int angka[kolom];
    for (a = 0; a<kolom; a++){
        scanf("%d",&isi);
        angka[a] = isi ;}
    for (a =0;a<kolom;a++){
        printf("%d ",angka[a] * (a + 1)); }    
}